sylt-common/src/value.rs

```diff
@@ -172,6 +160,115 @@ impl Value {
             Value::ExternFunction(slot) => slot + 2,
         }
     }
+
+    fn safe_fmt(
+        &self,
+        fmt: &mut std::fmt::Formatter<'_>,
+        seen: &mut HashSet<usize>
+    ) -> std::fmt::Result {
+        match self {
+            Value::Field(s) => write!(fmt, "<field .{}>", s),
+            Value::Ty(ty) => write!(fmt, "<type \"{:?}\">", ty),
+            Value::Blob(b) => write!(fmt, "<blob \"{}\">", b),
+            Value::Instance(_, v) => {
+                write!(fmt, "{} {{",
+                    if let Some(Value::String(name)) = v.borrow().get("_name") {
+                        name.as_str()
+                    } else {
+                        "?"
```

Comment:
Maybe the memory location? At least _something_ that is unique per instance.

Reply:
Currently - the "?" should be unreachable. Since we don't have such a thing as anonymous blobs. But maybe it's worth adding the pointer to differentiate instances of object - like Gustav suggests. Maybe it should be written after the type?
```
A (0xdeadbeef) { .a = 1 }
```
?

Reply:
Write unreachable or unimplemented